Makefile

```diff
@@ -94,4 +111,8 @@ help:
 	@echo '  the charm will be deployed in the default Juju environment.'
 	@echo '  If SERIES is not passed, "trusty" is used. Possible values are'
 	@echo -e '  "precise" and "trusty".\n'
+	@echo 'make publish-development - Push and publish the charm to the'
+	@echo -e '  development channel.\n'
+	@echo 'make publish-stable - Push and publish the charm to the'
+	@echo -e '  stable channel.\n'
```

Comment:
One reason for including these push/publish targets in the makefile is that it is a multistep process. Doing the clean first is important to ensure no build artifacts get pushed up. It seems like publishing to stable would be a candidate for removing potential errors.